### PR TITLE
add cohere embedding example

### PR DESCRIPTION
**Pull request by yiweny**

### CODE REVIEW

File: examples/ft_transformer_text.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import argparse
+import os
 import os.path as osp
 from typing import List
 
+import cohere
```
Comment (zechengz) [inline examples/ft_transformer_text.py:6]: Move this to the cohere embedding class? Now it requires user install cohere to run the text example.

**Review by zechengz — COMMENTED** [review]
Left some comment

**Comment by weihua916** [comment]
Thanks! How about we merge this with https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py, since both are using API?
and rename to `text_api_embeddings.py`?

File: examples/ft_transformer_text.py (hunk 1 above)
Comment (yiweny) [reply examples/ft_transformer_text.py:6]: Makes sense

**Comment by yiweny** [comment]
> Thanks! How about we merge this with https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py, since both are using API? and rename to `text_api_embeddings.py`?

But I want to put the experiment results we produced yesterday somewhere. Or we can put it in the readme?

**Comment by weihua916** [comment]
Got it, but I think it's cleaner to have well-organized example code according to their functionality. I don't think where you keep results is so important.

**Comment by yiweny** [comment]
> Got it, but I think it's cleaner to have well-organized example code according to their functionality. I don't think where you keep results is so important.

Resolved~

File: examples/openai_embedding.py [hunk 2]
```diff
@@ -28,21 +26,33 @@
 parser.add_argument('--lr', type=float, default=0.0001)
 parser.add_argument('--epochs', type=int, default=100)
 parser.add_argument('--seed', type=int, default=0)
+parser.add_argument('--service', type=str, default='cohere',
+                    choices=['openai', 'cohere'])
 parser.add_argument('--model', type=str, default='text-embedding-ada-002')
-parser.add_argument('--emb_size', type=int, default=1536)
 parser.add_argument('--openai_api_key', type=str, default=None)
+parser.add_argument('--cohere_api_key', type=str, default=None)
```
Comment (weihua916) [inline examples/openai_embedding.py:33]: nit: why not just have `api_key`?

File: examples/openai_embedding.py [hunk 3]
```diff
@@ -28,21 +26,33 @@
 parser.add_argument('--lr', type=float, default=0.0001)
 parser.add_argument('--epochs', type=int, default=100)
 parser.add_argument('--seed', type=int, default=0)
+parser.add_argument('--service', type=str, default='cohere',
+                    choices=['openai', 'cohere'])
 parser.add_argument('--model', type=str, default='text-embedding-ada-002')
```
Comment (weihua916) [inline examples/openai_embedding.py:31]: Can we drop model argument since it's special to openai?

**Review by weihua916 — APPROVED** [review]
Looks great. Logo as well :)

File: README.md [hunk 4]
```diff
@@ -183,6 +187,34 @@ for epoch in range(50):
...
+      <a href="https://platform.openai.com/docs/guides/embeddings">
+        <img src="docs/source/_figures/OpenAI_Logo.png" alt="OpenAI" width="100px"/>
+      </a>
+      <br /><a href="https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py">OpenAI Embedding Code Example</a>
+    </td>
+    <td align="center">
+      <a href="https://cohere.com/embeddings">
+        <img src="docs/source/_figures/cohere-logo.png" alt="Cohere" width="100px"/>
+      </a>
+      <br /><a href="https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py">Cohere Embed v3 Code Example</a>
```
Comment (zechengz) [inline README.md:207]: Rename the `openai_embedding.py` to `text_api_embeddings.py` as mentioned by Weihua and change both openai and cohere example path to that?

Comment (yiweny) [reply README.md:207]: I want to call it `llm_embeddings.py` if that's fine with you

Comment (zechengz) [reply README.md:207]: I am ok. Though it sounds too fancy to me 😂

**Pull request merged**